docs/core/preview3/tools/dotnet-build.md

```diff
@@ -23,33 +23,37 @@ dotnet-build -- Builds a project and all of its dependencies
 
 ## Synopsis
 
-`dotnet build [--help] [--output]  [--framework]  
-    [--configuration]  [--runtime] [--version-suffix]
-    [--build-profile]  [--no-incremental] [--no-dependencies]
-    [<project>]`
+```
+dotnet build <project>
+dotnet build [-h|--help]
+dotnet build [-o|--output] [-f|--framework] [-c|--configuration] [-r|--runtime]
+dotnet build [--version-suffix] [--no-incremental] [--no-dependencies]
+```
+
+
 
 ## Description
+The `dotnet build` command builds the project and its dependencies into a set of binaries. The binaries are the symbol files used for debugging (having a `*.pdb` extension) as well as the project's code in Intermediate Language (IL) with an `*.dll` extension. Additionally, a JSON file that lists out the dependencies of the application with the `*.deps` extension will be produced. Finally, a `runtime.config.json` file will be produced as well. This file specifies which shared runtime and version the built code will run against. 
```

Comment:
an dll -> a dll 

should these outputs be on a list? 

and what is the JSON file produced? Does it have a name?

runtime.config.json -> *.runtime.config.json?